### PR TITLE
Refactor/better debug

### PR DESCRIPTION
**Pull request by Phoenix500526**
Please briefly answer these questions:
Currently, some gRPC requests and responses consist of bytes and are not human-readable. I implemented the Display trait for these requests and responses so they can be output in a more developer-friendly format. The reason for implementing Display instead of Debug is that Prost automatically derives Debug for all requests and responses. This pr can reduce the log content size which mentioned in issue #516 .

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)
Implement Display trait for some requests and responses

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)
no

### CODE REVIEW

**Review by bsbds — APPROVED** [review]

File: crates/xline/src/server/auth_server.rs [hunk 1]
```diff
@@ -142,8 +142,8 @@ where
         &self,
         mut request: tonic::Request<AuthUserAddRequest>,
     ) -> Result<tonic::Response<AuthUserAddResponse>, tonic::Status> {
-        debug!("Receive AuthUserAddRequest {:?}", request);
         let user_add_req = request.get_mut();
+        debug!("Receive AuthUserAddRequest {}", user_add_req);
```
Comment (rogercloud) [inline crates/xline/src/server/auth_server.rs:146]: It's in the `debug` tracing call. Why do you use display instead of debugging?

Comment (rogercloud) [reply crates/xline/src/server/auth_server.rs:146]: Try to override this trait https://doc.rust-lang.org/std/fmt/trait.Debug.html

Comment (Phoenix500526) [reply crates/xline/src/server/auth_server.rs:146]: The prost will automatically derive the debug trait for all the requests and responses it generates.  So far, we cannot switch off this behavior for the prost crate. FYI: https://github.com/tokio-rs/prost/issues/334. So I have to override the Display trait for it.

**Review by bsbds — APPROVED** [review]

File: crates/xline/src/storage/kvwatcher.rs [hunk 2]
```diff
@@ -552,6 +552,19 @@ pub(crate) struct WatchEvent {
     compacted: bool,
 }
 
+impl std::fmt::Debug for WatchEvent {
+    #[allow(clippy::arithmetic_side_effects)]
```
Comment (rogercloud) [inline crates/xline/src/storage/kvwatcher.rs:556]: If it's affected by the macro, please fix it in the macro.

Comment (Phoenix500526) [reply crates/xline/src/storage/kvwatcher.rs]: Done

**Review by rogercloud — APPROVED** [review]

**Review by bsbds — APPROVED** [review]

**Pull request merged**